File: conditionals/oldassignment.py
#Santiaog Lagarde Time of Day
import time
current =time.time()
local_time= time.localtime(current)
hour = local_time.tm_hour
if hour >=5 & hour <=11:
    print(f"Good Morning.")
elif hour >11 & hour  <= 16:
    print(f"Good Afternoon")
elif hour >16 & hour <19:
    print("Good Evening.")
elif hour >19 & hour <=23:
    print("Good Night.")

File: conditionals/oldassignment.c
#include <stdio.h> 
#include <time.h> 
int hour;

int main ()
{
    time_t rawtime;
    struct tm *timeinfo; 

    time(&rawtime); 
    timeinfo = localtime (&rawtime); 
    printf ("tell me the hour in military time (like 1-23):\n"); 
    scanf("%d", &hour); 
    if (hour <  12 && hour >=5){
    printf("Good Morning."); 
    }else if (hour<=18 && hour >=12){
        printf("Good Afternoon"); 
    }else if (hour>=19 && hour <=21){
        printf ("Good Evening.");
    }else{
      printf("Good Night.");
    }
    
    return 0; 
}